online-workflow/src/main/java/eu/itesla_project/online/tools/PrintOnlineWorkflowPostContingencyLoadflowTool.java

```diff
@@ -88,104 +94,85 @@ public Command getCommand() {
+                        }
+                    }
+                } else
+                    System.out.println("\nNo post contingency loadflow data for workflow " + workflowId + " and state " + stateId);
+            } else if (line.hasOption("contingency")) {
+                String contingencyId = line.getOptionValue("contingency");
+                Map<Integer, Boolean> loadflowConvergence = onlinedb.getPostContingencyLoadflowConvergence(workflowId, contingencyId);
+                if (loadflowConvergence != null && !loadflowConvergence.keySet().isEmpty()) {
+                    try (TableFormatter formatter = createFormatter(formatterFactory, tableFormatterConfig, writer)) {
+                        Integer[] stateIds = loadflowConvergence.keySet().toArray(new Integer[loadflowConvergence.keySet().size()]);
+                        Arrays.sort(stateIds);
+                        for (Integer stateId : stateIds) {
+                            Boolean loadflowConverge = loadflowConvergence.get(stateId);
+                            printValues(formatter, stateId, contingencyId, loadflowConverge);
+                        }
+                    }
+                } else
+                    System.out.println("\nNo post contingency loadflow data for workflow " + workflowId + " and contingency " + contingencyId);
+            } else {
+                Map<Integer, Map<String, Boolean>> loadflowConvergence = onlinedb.getPostContingencyLoadflowConvergence(workflowId);
+                if (loadflowConvergence != null && !loadflowConvergence.keySet().isEmpty()) {
+                    try (TableFormatter formatter = createFormatter(formatterFactory, tableFormatterConfig, writer)) {
+                        Integer[] stateIds = loadflowConvergence.keySet().toArray(new Integer[loadflowConvergence.keySet().size()]);
+                        Arrays.sort(stateIds);
+                        for (Integer stateId : stateIds) {
+                            Map<String, Boolean> stateLoadflowConvergence = loadflowConvergence.get(stateId);
+                            if (stateLoadflowConvergence != null && !stateLoadflowConvergence.keySet().isEmpty()) {
+                                String[] contingencyIds = stateLoadflowConvergence.keySet().toArray(new String[stateLoadflowConvergence.keySet().size()]);
+                                Arrays.sort(contingencyIds);
+                                for (String contingencyId : contingencyIds) {
+                                    Boolean loadflowConverge = stateLoadflowConvergence.get(contingencyId);
+                                    printValues(formatter, stateId, contingencyId, loadflowConverge);
+                                }
+                            }
+                        }
+                    }
+                } else
+                    System.out.println("\nNo post contingency loadflow data for workflow " + workflowId);
+            }
+        }
```

Comment:
you're probably right;  take into account, though, that refactoring the existing code was not intended:  the committed changes were just meant to support -quickly-  because it was needed, the csv configurations.